BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -180,4 +170,15 @@
  **/ 
 - (void)setSharedLinkStorageDelegate:(id <BOXSharedLinkStorageProtocol>)delegate;
 
+/**
+ *  API base URLs.
+ **/
++ (NSString *)APIBaseURL;
++ (NSString *)OAuth2BaseURL;
++ (NSString *)APIAuthBaseURL;
```

Comment:
As discussed offline, a bit of a better way to differentiate APIAuthBaseURL from OAuth2BaseURL without having to dig into the code around them (whether that's in naming, comments, etc).